src/plot/series/abstract-series.js

```diff
@@ -51,7 +51,8 @@ const propTypes = {
   onNearestX: PropTypes.func,
   onNearestXY: PropTypes.func,
   style: PropTypes.object,
-  animation: AnimationPropType
+  animation: AnimationPropType,
+  stack: PropTypes.bool
 };
 
 const defaultProps = {
```

Comment:
probably better to set the default stack prop on abstract-series's defaultProps rather than on every other series

Reply:
That's what I did at first but I wasn't sure it would be inherited as I saw some series defining their own `defaultProps`. Will do !